.ci/packaging.groovy

```diff
@@ -73,124 +86,20 @@ pipeline {
             }
           }
         }
-        stage('Package') {
```

Comment:
Temporarily removed for testing purposes. It will not land in the target branch.